rma/models/res_partner.py

```diff
@@ -9,7 +9,7 @@ class ResPartner(models.Model):
 
     def _compute_rma_line_count(self):
         for rec in self:
-            rec.rma_line_count = len(rec.rma_line_ids)
+            rec.sudo().rma_line_count = len(rec.rma_line_ids)
```

Comment:
you can use compute_sudo in the field definition

Reply:
Do you have any idea why the compute_sudo is not working ? :S

Reply:
Hi @LoisRForgeFlow , I am afraid the compute_sudo = True does not work, I am not sure if it is because of the type of field is Integer. It just doesn't work :( So I gave up and put the sudo in the method again. Do you think we can move forward with the PR despite that?

Reply:
Finally I got the real issue. compute_sudo works.  The problem was the test calling the _compute_rma_line_count directly with a different user. Calling the compute in the test makes no sense so I removed that call. Now it should be green and ready to be merged.